OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -350,14 +350,14 @@ void GeometryPath::addInEquivalentForces(const SimTK::State& s,
             // add in the tension point forces to body forces
             if (mppo) {// moving path point location is a function of the state
                 // transform of the frame of the point to the base mobilized body
-                auto X_BF = mppo->getParentFrame().findTransformInBaseFrame();
+                auto X_BF = mppo->getBody().findTransformInBaseFrame();
```

Comment:
Agreed, especially since the functions seem to do the same thing. Even if `getBody()` actually returned the base frame (body) and not the parent frame, the function call is still to get the transform in the base frame.